king_phisher/client/windows/plugin_manager.py

```diff
@@ -357,22 +394,25 @@ def _plugin_install(self, model_row):
 					return
 		self._worker_thread_start(self._plugin_install_tsafe, catalog_model, repo_model, model_row, named_row)
 
+	def _dialog_error_tsafe(self, error_message):
```

Comment:
Can you please put this back? The reason why this was originally inlined instead of a method is that the title was always  `Failed To Install` but the method name makes it sound more flexible than that. That is to say that this method shouldn't be called elsewhere to show other errors that don't have to do with the installation process. This should either take the title as an argument or go back to being inlined using partial like it was.